app/models/article.rb

```diff
@@ -351,6 +352,22 @@ def update_score
                    spaminess_rating: BlackBox.calculate_spaminess(self))
   end
 
+  def liquid_tags_used(section = nil)
```

Comment:
Does this really need to live on the model? A caller of this method already has access to an article object, so unless you call this in many different places, it can live closer to where it's needed? `Article` and `User` area already very heavy on responsibilities, we should always consider if we want to add more to that.

Reply:
Nope - it doesn't! It can be removed. Right now it's currently being used for our "hacky" liquid tag permission setup. Once I refactor how that works in 2 more PRs, I'll come back and take this out.